src/yaxarrays/YAXArrays.jl

```diff
@@ -0,0 +1,36 @@
+import Base.copy
+
+function axes_names(cube::YAXArray)
+    return name.(cube.axes)
+end
+
+"""
+    sort(cube::YAXArray, axis_name::Symbol)
+
+Sorts axis labels of a given YAXArray datacube.
+"""
+function sort(cube::YAXArray, axis_name::Symbol)::YAXArray
```

Comment:
Not a big fan of the module/filename.

We already have an `io` folder, can we not move it there?

Reply:
Even with the YAXArray constructor?